datastore/client/datastore_client.go

```diff
@@ -1,4 +1,4 @@
-package datastore_client
+package client
```

Comment:
Same deal - recommend renaming this to be `datastore_service_client` so that when users use it in their code they won't get `client` package name collisions between the API service & datastore service

Reply:
Aside: oh wow, I had no idea I'd already committed a client here - that was dumb of me; I shouldn't have checked this in :P